src/ARToolkitNFT.ts

```diff
@@ -4,6 +4,15 @@ import Utils from './Utils'
 const UNKNOWN_MARKER = -1
 const NFT_MARKER = 0
 
+declare global {
+  interface Window {
+    artoolkitNFT: any;
```

Comment:
with this code artoolkitNFT is added to the Window DOM Module but not understand how to do the same for the global NodeJS Module, the code below has no effect...

Reply:
solved with https://github.com/webarkit/jsartoolkitNFT/pull/89/commits/f3d76587519ab85c44653fbf24c406f0914db0d2 following this stackoverflow article https://stackoverflow.com/questions/40743131/how-to-prevent-property-does-not-exist-on-type-global-with-jsdom-and-t